docs/docs/policy-language.md

```diff
@@ -578,7 +578,7 @@ result := {
 
 ### Multiple Expressions
 
-Rules are often written in terms of multiple expressions that contain references to documents. In the following example, the rule defines a set of arrays where each array contains an application name and a hostname of a server where the application is deployed.
+Rules are often written in terms of multiple expressions that contain references to documents. In the following example, the rule defines a set of arrays where each array [contains](#contains-keyword) an application name and a hostname of a server where the application is deployed.
```

Comment:
The `contains` keyword in this example defines what the set contains, not what each array contains. If you want to keep the link, you could perhaps move it up to "contain references" above. But I'd probably just remove it.